### PR TITLE
doc/user: upgrade some integrations

### PR DESCRIPTION
**Pull request by benesch**
Details in commit messages. The upshot is: upgrade Census to production support and Tableau to alpha support.

### Motivation

   * This PR updates documentation.

### Checklist

- [x] This PR has adequate test coverage / QA involvement has been duly considered.
- [x] This PR has an associated up-to-date [design doc](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/README.md), is a design doc ([template](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/00000000_template.md)), or is sufficiently small to not require a design.
  
- [x] If this PR evolves [an existing `$T ⇔ Proto$T` mapping](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/command-and-response-binary-encoding.md) (possibly in a backwards-incompatible way), then it is tagged with a `T-proto` label.
- [x] If this PR will require changes to cloud orchestration, there is a companion cloud PR to account for those changes that is tagged with the release-blocker label ([example](https://github.com/MaterializeInc/cloud/pull/5021)).
  
- [x] This PR includes the following [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/guide-changes.md#what-changes-require-a-release-note):
  - n/a

### CODE REVIEW

File: doc/user/content/integrations/_index.md [hunk 1]
```diff
@@ -202,23 +202,23 @@ The level of support for these tools will improve as we extend the coverage of `
...
+| Tableau            | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://help.tableau.com/current/pro/desktop/en-us/examples_postgresql.htm). Use the JDBC-based driver, which is included by default on Windows but requires an extra [driver download](https://www.tableau.com/support/drivers) on macOS and Linux. | [](#notify) |
+| Microsoft Power BI | {{< supportLevel researching >}} | Not supported yet. Subscribe via "Notify Me" to register interest.                                                                                                                                                                                                                                  | [](#notify) |
+| Mode Analytics     | {{< supportLevel researching >}} | Not supported yet. Subscribe via "Notify Me" to register interest.                                                                                                                                                                                                                                  | [](#notify) |
```
Comment (morsapaes) [inline doc/user/content/integrations/_index.md:221]:
```suggestion                                                                                                                                                                                                                       
```

Comment (morsapaes) [reply doc/user/content/integrations/_index.md:221]: Mode was acquired by Thoughtspot a couple of weeks ago.

File: doc/user/content/integrations/_index.md [hunk 2]
```diff
@@ -228,10 +228,10 @@ The level of support for these tools will improve as we extend the coverage of `
 
 ### Reverse ETL
 
-| Service   | Support level                    | Notes                                                                               |             |
-| --------- | -------------------------------- | ----------------------------------------------------------------------------------- | ----------- |
-| Census    | {{< supportLevel researching >}} | Not supported yet. Subscribe via "Notify Me" to register interest.                  | [](#notify) |
-| Hightouch | {{< supportLevel in-progress >}} | Connect using a [PostgreSQL source](https://hightouch.io/docs/sources/postgresql/). | [](#notify) |
+| Service   | Support level                    | Notes                                                                                                |             |
+| --------- | -------------------------------- | ---------------------------------------------------------------------------------------------------- | ----------- |
+| Census    | {{< supportLevel production >}}  | Connect using Census's [native Materialize source](https://docs.getcensus.com/sources/materialize).  |             |
```
Comment (morsapaes) [inline doc/user/content/integrations/_index.md:233]: I'm not really comfortable with this until we test the integration, but also don't strongly oppose. This is on @dehume's list for next week.

File: doc/user/content/integrations/_index.md [hunk 3]
```diff
@@ -202,23 +202,23 @@ The level of support for these tools will improve as we extend the coverage of `
...
+| Looker             | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://superset.apache.org/docs/databases/postgres/).                                                                                                                                                                                               | [](#notify) |
+| Google Data Studio | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://support.google.com/datastudio/answer/7288010?hl#how-to-connect-to-postgresql&zippy=%2Cin-this-article).                                                                                                                                      | [](#notify) |
+| Tableau            | {{< supportLevel alpha >}}       | Connect using the [PostgreSQL database driver](https://help.tableau.com/current/pro/desktop/en-us/examples_postgresql.htm). Use the JDBC-based driver, which is included by default on Windows but requires an extra [driver download](https://www.tableau.com/support/drivers) on macOS and Linux. | [](#notify) |
```
Comment (morsapaes) [inline doc/user/content/integrations/_index.md:219]: This reminds me I owe Bobby a review on #20030.

**Review by morsapaes — APPROVED** [review]

File: doc/user/content/integrations/_index.md (hunk 1 above)
Comment (benesch) [reply doc/user/content/integrations/_index.md]: Thanks, added.

File: doc/user/content/integrations/_index.md (hunk 2 above)
Comment (benesch) [reply doc/user/content/integrations/_index.md]: Ack, backed out for now.

File: doc/user/content/integrations/_index.md (hunk 3 above)
Comment (benesch) [reply doc/user/content/integrations/_index.md:219]: Oh, nice, didn't realize that was up. I'll rebase that PR on this one once I merge.

**Pull request merged**